apps/api/tests/test_guest.py

```diff
@@ -72,16 +72,17 @@ def test_guest_login_initiation(
     assert res.headers["Set-Cookie"].startswith("guest_confirmation=abcdef;")
 
 
-@patch("auth.guest_auth._get_existing_key", autospec=True)
-def test_requesting_login_when_previous_key_exists_causes_429(
-    mock_get_existing_key: AsyncMock,
-) -> None:
-    """Test that requesting to log in as guest when the user has an existing,
-    unexpired key causes status 429."""
-
-    mock_get_existing_key.return_value = "some-existing-key"
-    res = client.post("/login", data=SAMPLE_LOGIN_DATA)
-    assert res.status_code == 429
+# Commenting out this test to disable 429 from existing, unexpired key
```

Comment:
Could we update the test to reflect that in such a situation, the result should be being redirected to the verification and not having the `guest_confirmation` cookie modified?